src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -127,11 +128,11 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
               </S.ListItem>
             ))
           ) : (
-            <div className="dp-msj-error bounceIn">
+            <BoxMessage className="dp-msj-error bounceIn">
```

Comment:
This styled component should be named with prefix like `S.BoxMessage` I think we agreed on that some time ago.

Reply:
Same